### ISSUE TITLE
Some items not calculating correct EMC when ingredients have custom values

### ISSUE THREAD

**Issue by Havanacus** [sinkillerj/ProjectE#989]
Using ProjectE-1.7.10-PE1.8.0    Forge 10.13.4.1492
I've been adding EMC values to the base ingredients in Simply Jetpacks and Redstone Arsenal.  Everything in Simply Jetpacks calculated up flawlessly once I added the raw ingredients, but Redstone Arsenal is giving odd, inconsistent values.

For example, Fluxed Armor Plating is worth 19,370 and Flux-Infused Helm is simply 5 of these crafted together.  There are no alternate recipes, yet the Helm was calculated to be worth 213 EMC.
The kicker is that the values aren't even consistent.  The boots, which require 4 plates, are calculated at 2,318 EMC.  The Legging, which require 7 plates, are calculated at 135,590 EMC.  And the Chestplate, which requires 8 plates, is calculated at 4,636 EMC.

Restarting my Client and the reloadEMC command did not help.

**Comment by MaPePeR** [comment]
It might think the items are damaged and reducing the EMC value. 
Could you generate a pregen file and check what values were written into that for these items?

**Comment by Havanacus** [comment]
Redstone Arsenal seems to use a separate Durability and Charge value, so the items always have a full 400,000 durability.  In any case, I spawned an uncharged helmet and charged it, the EMC value remained constant.

I generated a pregen file and the values in it appear to be correct.
"RedstoneArsenal:armor.helmetFlux|0": 96850
"RedstoneArsenal:armor.plateFlux|0": 154960
"RedstoneArsenal:armor.legsFlux|0": 135590
"RedstoneArsenal:armor.bootsFlux|0": 77480
